{"bug_class": "PROVER", "kind": "ERROR", "bug_type": "RESOURCE_LEAK", "qualifier": "resource of type `org.simpleframework.transport.connect.SocketConnection` acquired to `socketConnection` by call to `new()` at line 41 is not released after line 42.", "severity": "HIGH", "visibility": "user", "line": 42, "column": -1, "procedure": "void SimpleServerWrapper.start(int,SSLContext,boolean)", "procedure_id": "net.codestory.http.internal.SimpleServerWrapper.start(int,javax.net.ssl.SSLContext,boolean):void.c72bb5bb08d33bcf63a413a11d9b803f", "procedure_start_line": 40, "file": "src/main/java/net/codestory/http/internal/SimpleServerWrapper.java", "bug_trace": [{"level": 0, "filename": "src/main/java/net/codestory/http/internal/SimpleServerWrapper.java", "line_number": 40, "column_number": -1, "description": "start of procedure start(...)"}, {"level": 0, "filename": "src/main/java/net/codestory/http/internal/SimpleServerWrapper.java", "line_number": 41, "column_number": -1, "description": "Taking false branch"}, {"level": 0, "filename": "src/main/java/net/codestory/http/internal/SimpleServerWrapper.java", "line_number": 42, "column_number": -1, "description": "Skipping connect(...): unknown method"}], "key": "SimpleServerWrapper.java|start|RESOURCE_LEAK", "node_key": "8a01c05f04ccba1101dfb3237187637c", "hash": "3d5f3465df02fcec36801d4e170709ba", "bug_type_hum": "Resource Leak", "censored_reason": ""}